datafusion-cli/src/catalog.rs

```diff
@@ -237,7 +236,7 @@ fn substitute_tilde(cur: String) -> String {
 mod tests {
     use super::*;
 
-    use datafusion::catalog::schema::SchemaProvider;
+    use datafusion::catalog_api::SchemaProvider;
```

Comment:
why does it called catalog-api but not the same name as crate?

Reply:
let's merge this thread with https://github.com/apache/datafusion/pull/11516#discussion_r1685354303